aten/src/ATen/native/cuda/LegacyDefinitions.cpp

```diff
@@ -59,26 +59,6 @@ Tensor & masked_scatter__cuda(Tensor& self, const Tensor & mask, const Tensor &
   }
 }
 
-Tensor masked_select_cuda(const Tensor & self, const Tensor & mask) {
```

Comment:
That cleanup is nice, can you also delete corresponding THC files?

Reply:
Done